Twittfake/src/components/Navbar/Navbar.jsx

```diff
@@ -7,34 +7,17 @@ import {
   faUser,
 } from "@fortawesome/free-solid-svg-icons";
 import { Link } from "react-router-dom";
-import { useEffect, useState } from "react";
+import ScrollHandler from "../../functions/scrollHandler";
 
 const Navbar = () => {
   const userLogged = sessionStorage.getItem("username");
   const navbarIcons = [faPlus, faMagnifyingGlass];
-  const [showNavbar, setShowNavbar] = useState(true);
-  const [lastScrollPos, setLastScrollPos] = useState(0);
-
   const routeToLoginOrProfile = userLogged ? "/Profile" : "SignIn";
 
-  const handleScroll = () => {
-    const currentScrollPos = window.scrollY;
-    if (currentScrollPos > lastScrollPos) {
-      setShowNavbar(false);
-    } else setShowNavbar(true);
-    setLastScrollPos(currentScrollPos);
-  };
-
-  useEffect(() => {
-    window.addEventListener("scroll", handleScroll);
-
-    return () => {
-      window.removeEventListener("scroll", handleScroll);
-    };
-  });
+  const showElement = ScrollHandler();
 
   return (
-    <div className={`navbar ${showNavbar ? "" : "hidden"}`}>
+    <div className={`navbar${showElement ? "--hidden" : ""}`}>
```

Comment:
U can also check `classNames` from ddtvn-waiter repo, `helper.js`. We're using this one for handling classnames which depends on variables/props. 

```
export function classNames(array) {
    return array.filter((value) => value).join(' ');
}
```

example of use: 
```
<figure
            className={classNames([
                'stories-list-content',
                'stories-list-article-content',
                isPwa && 'stories-list-content--pwa',
            ])}>
            <Link href={url}>
            [...]
```

Reply:
done